index.html

```diff
@@ -0,0 +1,109 @@
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Mint Sign Up Form</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        
+        <!--CSS -->
+        <link rel="stylesheet" href="css/style.css">
+        <!--font Awsome Icons CDN-->
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.1/css/all.min.css">
+        
+    </head>
+    <body>
+    <!--Main container div-->
+
+    <div class="container">
+
+       <!--Header Section-->
+        <div class="header">
+            <span class="navbar-text">
+                <p>Already have an Intuit account?</p>
+                <a href="#">I forgot my User ID or Password</a> 
+            </span>
+            <span class="navbar-btn">
+                <button class="signin-btn"><i class="fa fa-lock" aria-hidden="true" style="color: white;">   Sign In</i></button>
+            </span>
+        </div>
+        <!--End of Header Section--> 
+
+        <!--form section-->
+        <div class="main-section">
+            <form >
+                <div class="container-logo">
+                    <img class="img-scl" src="images/intuit-logo.png">
+                </div>
+                <ul class="logos">
+                        <li><a href="#"><img src="images/mint-logo.png"></a></li>
+                        <li><a href="#"><img src="images/logo-quickbooks.png"></a></li>
+                        <li><a href="#"><img src="images/logo-turbotax.png"></a></li>
+                </ul>
+                <fieldset class="signup-form">
+                    <fieldset class="form-text">
+                        <p class="form-title">Create an Intuit account</p>
+                        <p style="font-size: 15px;">One account for everything Intuit, including Mint.</p>
+                        <a href="#">Learn more</a>
+                    </fieldset>
+                <fieldset class="space">
+                    <label for Email>Email address</label>
+                    <div>
+                        <input class="input-text" type="email">
+                    </div>
+                </fieldset>
+                <fieldset class="space">
+                    <label for number>Phone (recommended)</label>
+                    <div>
+                        <input class="input-text" type="tel">
+                    </div>
+                    <p>Standard call, messaging or data rates may apply.</p>
+                </fieldset>
+                <fieldset class="space">
+                    <label for password>Password</label>
+                    <div>
+                        <input class="input-text" type="password">
+                    </div>
+                </fieldset>
+                <span class="navbar-btn">
+                    <button class="submit-btn"><i class="fa fa-lock" aria-hidden="true" style="color: white;"> Create Account</i></button>
```

Comment:
- [ ] **[MANDATORY]** For good practices in CSS, do not use inline styling, put it in your css file
- [ ] **[OPTIONAL]** When you use a button in a form, it should have the type attribute submit to work in a  functional form that submits data to the server, it will look like this: `<button type="submit" class="submit-btn">`